This test tries to install cmake from PyPI, but its compilation fails.

--- scikit_build-0.18.1/tests/test_setup.py.orig
+++ scikit_build-0.18.1/tests/test_setup.py
@@ -327,6 +327,7 @@
 @pytest.mark.xfail(
     sys.platform.startswith("cygwin"), strict=False, reason="Cygwin needs a release of scikit-build first"
 )
+@pytest.mark.skip(reason="always fails: https://github.com/scikit-build/scikit-build/pull/983")
 def test_setup_requires_keyword_include_cmake(mocker, capsys):
     mock_setup = mocker.patch("skbuild.setuptools_wrap.setuptools.setup")
 
